serverless-chatbot/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "serverless-chatbot",
+  "version": "1.0.0",
+  "description": "Example bot implementation that uses Beekeeper API webhooks",
+  "main": "handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "beekeeper": "^0.4.1"
```

Comment:
0.4.2 ?

Reply:
From what I can see the latest published version is 0.4.1
![image](https://user-images.githubusercontent.com/9937780/70731346-d7750080-1d06-11ea-9882-8857df32d3c3.png)